src/util.ts

```diff
@@ -280,3 +297,36 @@ export const addHashToBaseline = (hash: string, filepath: string): void => {
     ignoreMessages: baselineErrorsFile.meta.ignoreMessages
   })
 }
+
+export const toGitLabOutputFormat = (
+  errorSummaryMap: ErrorSummaryMap,
+  specificErrorMap: SpecificErrorsMap,
+  errorOptions: ErrorOptions
+): string => {
+  const result: GitLabErrorFormat[] = []
+
+  for (const [key, error] of errorSummaryMap.entries()) {
+    const specificErrors = getSpecificErrorsMatchingSummary(
+      error,
+      specificErrorMap,
+      errorOptions
+    )
+
+    const specificError: SpecificError = specificErrors[0] || {}
+
+    result.push({
+      description: error.message || 'Unknown error message',
```

Comment:
The `message` in `specificError` is not optional. Therefore we do not need a fallback for it.

Reply:
Yes, that makes sense and has been removed.